### PR TITLE
#1387 Update tooltips to show overflow in textfields and table headers

### PR DESCRIPTION
**Pull request by matthoward366**
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

File: canvas_modules/harness/cypress/integration/canvas/tips.js [hunk 1]
```diff
@@ -269,58 +275,15 @@ describe("Test tip location adjusted based on boundaries of browser", function()
...
+				cy.verifyTip(container, "visible", "Include or discard rows", "bottom");
+			});
 		cy.clickAtCoordinatesInCommonProperties(300, 155);
-		cy.verifyTipForLabelIsVisibleAtLocation(
-			"Modeler CLEM Condition Expression", "bottom", "Enter a boolean expression to use for filtering rows"
-		);
-	});
-});
-
-describe("Test if tips show up for the summary table values", function() {
```
Comment (matthoward366) [inline canvas_modules/harness/cypress/integration/canvas/tips.js:281]: These were moved to a properties specific suite.

File: canvas_modules/common-canvas/src/common-properties/components/virtualized-table/virtualized-table.jsx [hunk 2]
```diff
@@ -234,12 +237,6 @@ class VirtualizedTable extends React.Component {
 			</span>);
 		}
 
-		const tooltip = columnData.headerLabel
```
Comment (matthoward366) [inline canvas_modules/common-canvas/src/common-properties/components/virtualized-table/virtualized-table.jsx:237]: This didn't see a case where we needed this check.

File: canvas_modules/harness/cypress/support/properties/properties-verification-cmds.js [hunk 3]
```diff
@@ -60,43 +60,73 @@ Cypress.Commands.add("verifyPropertiesFlyoutDoesNotExist", () => {
...
 /** Verify the tooltip over the given text is 'visible'
-* @param label: label of the container shown in the UI
+* @param container: the container shown in the UI
+* @param visible: "visible" or "hidden"
 * @param text: text shown in the tip
+* @param tipLocation: the location of the tooltip
 */
-Cypress.Commands.add("verifyTipForLabelIsVisibleAtLocation", (label, tipLocation, text) => {
-	// Verify the tip for label "Mode" is visible on the "top" with text "Include or discard rows"
-	cy.getControlContainerFromName(label)
```
Comment (matthoward366) [inline canvas_modules/harness/cypress/support/properties/properties-verification-cmds.js:68]: Most of this was duplicate so I created 1 method to handle tooltip verification.

**Comment by nmgokhale** [comment]
- Summary panel shows tooltip without ellipsis.
- This is an edge case. When I resize the column, header shows ellipsis with tooltip. But the text input cell doesn't show tooltip.

https://user-images.githubusercontent.com/25124000/229212315-868ae4cb-bd3d-4a89-89fe-e06d0156bc00.mov

**Comment by matthoward366** [comment]
Thanks for catching that.  The 2nd issue wasn't part of this fix.  That's a numberfield which we didn't add tooltips for. It's probably ok since in most cases users don't have that large of numbers.

**Comment by matthoward366** [comment]
I pushed a change for the summary panel.

**Review by nmgokhale — APPROVED** [review]

**Review by nmgokhale — APPROVED** [review]

**Pull request merged**